extension/src/fetch.ts

```diff
@@ -41,9 +41,10 @@ export async function fetchAndUpdate(ctx: ExtensionContext, prompt = true) {
 
   if (shouldUpdate) {
     const config = workspace.getConfiguration()
-    config.update('explorer.fileNesting.enabled', true, true)
+
```

Comment:
We should check if the `enabled` is explicitly set, and update when it's not.

Reply:
i wont recommend that because
1- if the user have installed this ext, he knows that this requires the config to be enabled
2- if the user chose to disable the config but still keep the ext, then we should respect that

what u suggest will leave the user with one of 2 options
- disable the ext to stop this behavior of force enabling the config on each update
- remove the ext completely

Reply:
The 1. sounds not very true to me. In general, users might not need to know what's the exact config of this feature is about, they should be able to install this ext and then get it working.

2. is what we want to solve and I agree. We should check if `explorer.fileNesting.enabled` is set (either `true` or `false`, we could respect the user's config). But if not (null), it means this setting as never be set, we should set it to `true`.

Reply:
> users might not need to know what's the exact config of this feature is about, they should be able to install this ext and then get it working.

in this case adding info to the readme is more appropriate, to educate the user about the feature and what it does instead of relying on the ext which is kinda misleading.